kubernetes-docs/overview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: vSphere Cloud Provider
+---
+
+## Overview
+Containers are stateless and ephemeral but applications are stateful and need persistent storage. vSphere adds this persistent storage support to Kubernetes through interface called Cloud Provider. Cloud provider is an interface which helps in extending Kubernetes with cluster of instances managed by virtualisation technologies, public/private cloud platforms and required networking for these instances. 
+ 
+ ![Image](images/vSphere.png)
+ 
+ 
+Kubernetes cloud provider is an interface to integrate various nodes (i.e. hosts), load balancers and networking routes. This interface allows extending Kubernetes to use various cloud and virtualisation solutions as base infrastructure to run on. 
+ 
+Cloud provider provides following interfaces to effectively integrate cloud platforms:
+
+* **Instances** - interface for virtual machine management
+* **Load Balancers** - interface to integrate with load balancer provided by cloud platform
+* **Routes** - interface to add new routing rules of cloud platform
+* **Zones** - integrate with zones if implemented by cloud platform
+
+
+## vSphere Storage Concepts
```

Comment:
I have made vSAN etc. as bullet points. A section will have to be list of paragraphs as in case of any docs, however we can always add bullet points like you mentioned.